### PR TITLE
Add basic UI 👨‍💻 

### PR DESCRIPTION
**Pull request by tsheporamantso**
## Add basic UI 👨‍💻 
On this milestone 🗿, I've managed to implement the following 🧠 
- [ ] Add `app.rb` and `main.rb` files 📁 as entry points  ✔️ 
- [ ] Defined the following methods on class `App` :
    ➕ `book_list` method that lists all books ✔️ 
    ➕ `people_list` method that lists all people ✔️
    ➕ `person_create` method that prompts a user whether to create an object of  `student` or `teacher` ✔️ 
    ➕ `teacher` and `student` methods that prompt a user to actually create an object of either `teacher` or `student` ✔️ 
    ➕ `new_book` method that prompts a user to create a book object ✔️ 
    ➕ `new_rental` method that prompts a user to create an object of renting a book ✔️
    ➕ `rental_list` method that prompts a user to insert an ID of a person and return a list of book(s) rented✔️
-  [ ]  Defined the following methods on the class `Main`
    ➕ `display_menu` method that presents the user with a list of options ✔️
    ➕ `options` method that allows the user to select  options ✔️
    ➕ `main` method that displays welcome text to the user, loops through the `display_menu` and `options` methods and also 
          terminates the app when the user selects option 7. ✔️ 
- [ ] To test the app run the following command `ruby main.rb` on the terminal and follow the prompts 🧪

### CODE REVIEW

File: app.rb
Comment (Efiamotu-1) [inline app.rb:1]: - [ ] Your application works fine i was pretty impressed as you followed all of the project requirements from the provided [feature video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). Great Job :+1: :sparkles: 
- [ ] However while going thrugh your readme i discovered a broken link under your `contributing section` for the `issues page` link. It is unprofessional to have broken links in your readme so i kindly suggest you fix appropriately and resubmit for approval :+1:

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @tsheporamantso, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb
Comment (tsheporamantso) [reply app.rb:1]: Issue resolved, "excuse the pun 😆" Thank you very much.

**Review by Bantealem — APPROVED** [review]
# Status: Approved
Hi @tsheporamantso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] Correct Git flow
- [x] Professional Readme
- [x] Descripitive PR Summary
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**